packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1086,14 +1143,16 @@ class PersistentBottomSheetController<T> extends ScaffoldFeatureController<_Pers
 class _ScaffoldScope extends InheritedWidget {
```

Comment:
These changes to `_ScaffoldScope` aren't needed anymore.

Reply:
Reverted the changes